src/components/TransferOption.js

```diff
@@ -13,36 +14,61 @@ export const TransferOption = ({
     onDoubleClick,
     value,
     icon,
+    active,
+    tooltipText,
+    dataTest,
 }) => {
-    return (
-        <div data-value={value} className="wrapper">
-            <div
-                className={cx('chip', { highlighted, disabled, selected })}
-                onClick={event => {
-                    if (disabled) return
-                    onClick({ label, value }, event)
-                }}
-                onDoubleClick={event => {
-                    if (disabled) return
-                    onDoubleClick({ label, value }, event)
-                }}
-            >
-                <span className="icon">{icon}</span>
-                <span className="label">{label}</span>
-            </div>
-
+    const renderContent = () => (
+        <div
+            className={cx('chip', {
+                highlighted,
+                disabled,
+                selected,
+                inactive: active !== undefined && !active,
+            })}
+            onClick={event => {
+                if (disabled) return
+                onClick({ label, value }, event)
+            }}
+            onDoubleClick={event => {
+                if (disabled) return
+                onDoubleClick({ label, value }, event)
+            }}
+            data-test={`${dataTest}-content`}
+        >
+            <span className="icon">{icon}</span>
+            <span className="label">{label}</span>
             <style jsx>{styles}</style>
         </div>
     )
+
+    return (
+        <div data-value={value} data-test={dataTest} className="wrapper">
+            {tooltipText ? (
+                <Tooltip
+                    key={`${value}`}
+                    content={tooltipText}
+                    placement={'top-start'}
```

Comment:
I cannot find `top-start` as possible value for placement in `Tooltip`, but `Popper` which is where the prop is passed ultimately, has it.
I assume it uses `top` but is there any proptype warning in the console?

Reply:
There is a warning in the console (screenshot). @cooper-joe will add the `top-start` to the proptypes at some point. For now it still works, as the prop gets passed to `Popper` as you mentioned above, but we'll have to live with the warning for a bit though.

![image](https://user-images.githubusercontent.com/12590483/108191058-fcd37a80-7112-11eb-937e-ebd0e6597b37.png)